CPAC/resources/configs/pipeline_config_benchmark-ANTS.yml

```
@@ -73,13 +73,6 @@ anatomical_preproc:
   acpc_alignment:
     T1w_brain_ACPC_template: $FSLDIR/data/standard/MNI152_T1_1mm_brain.nii.gz
 
```

I think we want to keep

```suggestion

  brain_extraction:
    run: On
```

here?